cogs5e/models/automation/effects/attack.py

```diff
@@ -125,9 +128,11 @@ def run(self, autoctx):
             else:
                 formatted_d20 = f'1d20{reroll_str}'
 
-            to_hit_message = 'To Hit'
+            to_hit_message = '**To Hit**:'
             if ac:
-                to_hit_message = f'To Hit (AC {ac})'
+                to_hit_message = f'**To Hit (AC {ac})**:'
+            if force_roll:
+                to_hit_message = '**To Hit**: Forced Roll!'
```

Comment:
If the `-ac` arg is provided here we're losing that information by overwriting `to_hit_message`